src/schemas/v1/incentive.ts

```diff
@@ -58,6 +59,13 @@ export const API_INCENTIVE_SCHEMA = {
       required: ['type'],
       additionalProperties: false,
     },
+    items: {
+      type: 'array',
```

Comment:
I'm having a hard time imagining how this will be applied to the situation. I'm familiar with how multiple heat pump incentives appear to be duplicates in our embed and similarly with weatherization incentives. 

Do you have an illustrative example of how this will be used? Are you thinking of grouping similar items in the client side code in embed? As of know, it looks like we are just adding some data redundancies wit hteh type property, but I'm sure I'm just missing something.

Reply:
I wrote up my thinking on this a few weeks ago here: https://www.notion.so/rewiringamerica/Granular-items-or-sub-items-274423aff1cd42f880367f13510500ae

The tl;dr is that I do anticipate some amount of grouping logic in the frontend, for incentives with multiple items. E.g. if an incentive has `items: [ductless_heat_pump, ducted_heat_pump]`, there would be some logic that recognizes that all the items are types of heat pump and would use the headline `$X off a heat pump`.

I'm not sure what you mean by the type property -- if you mean the `type` property of the `item` property, that's because I'm going to remove `item` altogether once this is rolled out and the embed and PEP are switched over to use `item`.

Reply:
I must have missed this. I just checked it out though. Also, thanks for the TLDR.